GuessTheMovie/GuessTheMovie.vcxproj

```diff
@@ -71,9 +71,13 @@
   <PropertyGroup Label="UserMacros" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
     <LinkIncremental>true</LinkIncremental>
+    <IncludePath>D:\boost_1_63_0;D:\boost_1_63_0\boost;$(IncludePath)</IncludePath>
+    <LibraryPath>D:\boost_1_63_0\stage\lib;$(LibraryPath)</LibraryPath>
```

Comment:
Please, let everyone know that they will have to install boost at the same place as you did!